main.go

```diff
@@ -51,12 +56,25 @@ func main() {
 		"API like targets or configuration. NOTE: \"all\" matching paths like \"/\" or \"\" and regex are not allowed.")
 	flagset.BoolVar(&errorOnReplace, "error-on-replace", false, "When specified, the proxy will return HTTP status code 400 if the query already contains a label matcher that differs from the one the proxy would inject.")
 
+	flagset.StringVar(&header, "header", "", "(Optional) An HTTP header to get the label value from.")
```

Comment:
The flag name header is duplicated and causes a panic

```go
panic: ./prom-label-proxy flag redefined: header

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc00031e420, {0x9a9288, 0xc000313670}, {0x8fec55, 0x6}, {0x91a2ac, 0x36})
        flag/flag.go:879 +0x2f4
flag.(*FlagSet).StringVar(...)
        flag/flag.go:762
main.main()
        main.go:59 +0x4d9
```

Reply:
Sorry, fixed. I only use this as a library, so I didn't notice.